src/sentry/tasks/post_process.py

```diff
@@ -216,10 +218,15 @@ def process_snoozes(group):
     """
     from sentry.models import GroupSnooze, GroupStatus
 
-    try:
-        snooze = GroupSnooze.objects.get_from_cache(group=group)
-    except GroupSnooze.DoesNotExist:
-        return False
+    key = "postprocess_gs:1:%s" % (group.id)
+    snooze = cache.get(key)
+    if not snooze:
+        try:
+            snooze = GroupSnooze.objects.get(group=group)
+            cache.set(key, snooze, 3600)
+        except GroupSnooze.DoesNotExist:
+            cache.set(key, 0, 60)
```

Comment:
Separately from my concerns, I don't think this will work as is. You're setting the key to `0` here. So when you fetch it, you'll get back `0`, which evaluates to `False`, so we'll just hit the database again.